src/controllers/admin/changeStatus.js

```diff
@@ -0,0 +1,24 @@
+const db = require('../../database/db');
+
+const changeStatus = async (request, response) => {
+
+    const {status} = request.body;
+    const {id} = request.params;
+
+    try{
+        const query = 'UPDATE requests SET status = $1 WHERE id =$2';
```

Comment:
You should first of all check if the request exists, before attempting to update it.